x/auctions/client/cli/query.go

```diff
@@ -0,0 +1,84 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/evmos/evmos/v19/x/auctions/types"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd returns the cli query commands for the auctions module.
+func GetQueryCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the auctions module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(
+		GetAuctionInfo(),
+		GetParamsCmd(),
+	)
+	return cmd
+}
+
+// GetAuctionInfo queries the current auction information
+func GetAuctionInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "info",
+		Short: "Get current auction info",
+		Long:  "Get current auction information.",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			res, err := queryClient.AuctionInfo(cmd.Context(), &types.QueryCurrentAuctionInfoRequest{})
+			if err != nil {
+				return err
+			}
+
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+	return cmd
+}
+
+// GetParamsCmd queries the auctions module params
+func GetParamsCmd() *cobra.Command {
```

Comment:
this is not a command:

```suggestion
// GetParams queries the auctions module params
func GetParams() *cobra.Command {
```

Reply:
I mean, it is a flag so pls update the name here or on the other flag for consistency